### ISSUE TITLE
Clarify aria-level and native heading elements

### ISSUE THREAD

**Issue by cblouch** [w3c/aria#1166]
In https://w3c.github.io/aria/#heading the prose talks about “heading elements” and that aria-level is needed to ensure “headings are organized into a logical outline”. This sounds like not just elements with a role of heading would require an aria-level. While it makes sense that a div with role=heading should be required to specify aria-level, it should be clarified that native semantics of an H1 do not require this. Likewise, if the author were to add role=heading to an H1, it seems like an aria-level should not be required.

**Comment by JAWS-test** [comment]
I don't see a problem here. 
* According to the specification, an HTML heading should not get role=heading (is redundant)
* This is only about ARIA headlines and they need aria-level
* If I use (contrary to the specification) role=heading on the HTML element for headings, the aria-level according to [HTML AAM](https://www.w3.org/TR/html-aam-1.0/) is implicit

**Comment by scottaohara** [comment]
Agree that I don't think there's an issue here, but I do think there's some room for more general cleanup here.

Will submit a PR with an update that should help to clear this up anyway.
